changelog/next_version.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -ueo pipefail
```

Comment:
Also use `set -e`?

Reply:
It's already there: `set -ueo` includes `-e` :)

Reply:
oh ok thx!